array-methods-basic/basic-find.js

```diff
@@ -6,16 +6,22 @@
 5. If it is not case insensitive, then make it!
  */
 
-var WORD_TO_FIND = 'Lorem';
-
+const WORD_TO_FIND = 'Lorem';
+const findMessages = function (messages) {
```

Comment:
Almost correct! Remember that .find returns first occurrence of the thing you're looking for in the array. So you don't need to return result[0].
You can use .toLowerCase() to make it case-insensitive.
With the extra parameter of the word to find I'd do it like this:
```javascript
const findMessages = function (messages, word) {
    const WORD_TO_FIND = word ? word : "Lorem"
    const result = messages.find((message)=>{
        return message.text.toLowerCase().match(WORD_TO_FIND.toLowerCase());
    })
    return result;
};
```